lib/stages/testDataGen.cpp

```diff
@@ -214,7 +279,7 @@ void testDataGen::main_thread() {
         frame_id_abs += 1;
         if (num_frames >= 0 && frame_id_abs >= num_frames) {
             INFO("Frame ID greater than the no. of frames");
-            exit_kotekan(ReturnCode::CLEAN_EXIT);
+            // exit_kotekan(ReturnCode::CLEAN_EXIT);
```

Comment:
I think we can just remove this entirely.  And also the `INFO` statement above.  Or at least change it to something like "Reached maximum number of frames, stopping data generation".

Reply:
Done